disco/util/serializer.py

```diff
@@ -31,8 +31,9 @@ def pickle():
 
     @classmethod
     def loads(cls, fmt, raw):
+        import yaml
         loads, _ = getattr(cls, fmt)()
-        return loads(raw)
+        return loads(raw, Loader=yaml.FullLoader)
```

Comment:
This change is incorrect and would break anyone using JSON files. What was the warning?

Reply:
Something about it being dangerous to load a yaml file without a loader specified.

Reply:
If it needs arguments passed now you will have to build something into this class that looks up and passes arguments based on the serializer format. Likely that `FORMATS` can just turn into a dict w/ options or lambda that does the calling.